### PR TITLE
Cleanup 2D

### PR DESCRIPTION
**Pull request by winterheart**
Removed M68K assembler code, formatting

### CODE REVIEW

File: src/Libraries/2D/Source/Flat8/fl8chfl8.c [hunk 1]
```diff
@@ -24,40 +24,41 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-	dst = grd_bm.bits + y*grd_bm.row + x+bm->w-1;
-	while (h--) {
-	w = bm->w;
-	while (w--)
-	 *dst-- = cl[*src++];
-	src += bm->row-bm->w;
-	dst += grd_bm.row+bm->w;
-	}
+void flat8_clut_hflip_flat8_ubitmap(grs_bitmap *bm, short x, short y,
+                                    uchar *cl) {
```
Comment (Interrupt) [inline src/Libraries/2D/Source/Flat8/fl8chfl8.c]: I don't like how short this formatting is wrapping things - what arguments are you running when doing the code cleanup?

Comment (winterheart) [reply src/Libraries/2D/Source/Flat8/fl8chfl8.c]: clang-format with default settings

Comment (winterheart) [reply src/Libraries/2D/Source/Flat8/fl8chfl8.c]: There 3 types of BracketAlignmentStyle https://clang.llvm.org/docs/ClangFormatStyleOptions.html , which would be better?

Comment (Interrupt) [reply src/Libraries/2D/Source/Flat8/fl8chfl8.c]: I think the ColumnLimit is what bugs me, 80 seems super narrow and makes me scroll way too much. Looking Glass seemed to not mind wide lines, maybe we can bump up to 120 or 130?

Comment (winterheart) [reply src/Libraries/2D/Source/Flat8/fl8chfl8.c]: Implemented .clang-format style with column-width 120.

**Comment by Interrupt** [comment]
Could we run the 120 format clang on this as well?

**Comment by winterheart** [comment]
Done.

**Pull request merged**